projects/igniteui-angular/src/lib/grids/common/grid.interface.ts

```diff
@@ -63,6 +65,7 @@ export interface CellType {
     grid: GridType;
     id?: { rowID: any; columnID: number; rowIndex: number };
     cellID?: any;
+    errors?: ValidationErrors;
```

Comment:
Oh almost forgot - because the errors are getters only, do mark them as readonly in the interface